python/src/iceberg/types.py

```diff
@@ -18,7 +18,7 @@
 from typing import Optional
 
 
-class Type(object):
```

Comment:
Can you remove unnecessary changes like these? We should do this in a separate PR to keep this one small.

Reply:
SG. will send a small PR for those.